app/helpers/users_helper.rb

```diff
@@ -1,2 +1,88 @@
 module UsersHelper
+  def self.default_content
+    stories = []
+    stories.push(Story.where(subscription_id: 1))
+    stories.push(Story.where(subscription_id: 2))
+    stories.push(Story.where(subscription_id: 3))
+    stories.flatten!
+    return stories
+  end  
+
+  def self.user_content(user)
+    stories = []
+    user.subscriptions.each do |subscription|
+      subscription.stories.each do |story|
+        stories.push(story)
+      end
+    end
+    return stories
+  end
+
+  def self.twitter_subscription_info(user_name)
+    twitter = Seemore::Application.config.twitter
+    tweets = twitter.user_timeline(user_name)
+    uid = tweets[0].user.id
+    provider = "twitter"
+    username = tweets[0].user.screen_name
+    avatar_url = tweets[0].user.profile_image_url
+    return tweets, uid, provider, username, avatar_url
+  end
+
+  def self.instagram_subscription_info(user_name)
+    results = HTTParty.get("https://www.instagram.com/#{user_name}/media/")
+    results = results["items"]
+    uid = results[0]["user"]["id"]
+    provider = "instagram"
+    username = user_name
+    avatar_url = results[0]["user"]["profile_picture"]
+    return results, uid, provider, username, avatar_url
+  end
+
+  def self.vimeo_call_videos(vimeo_user)
+    vimeo_env = ENV["VIMEO_ACCESS_TOKEN"]
+    return HTTParty.get("https://api.vimeo.com/users/#{vimeo_user}/videos", headers: {"Authorization" => "bearer #{vimeo_env}", 'Accept' => 'application/json' }, format: :json).parsed_response
+  end
+
+  def self.vimeo_subscription_info(vimeo_user)
+    vimeo_env = ENV["VIMEO_ACCESS_TOKEN"]
+    video_results = HTTParty.get("https://api.vimeo.com/users/#{vimeo_user}/videos", headers: {"Authorization" => "bearer #{vimeo_env}", 'Accept' => 'application/json' }, format: :json).parsed_response
+    user_results = HTTParty.get("https://api.vimeo.com/users/#{vimeo_user}", headers: {"Authorization" => "bearer #{vimeo_env}", 'Accept' => 'application/json' }, format: :json).parsed_response
+    provider = "vimeo"
+    uid = vimeo_user
+    username = user_results["name"]
+    avatar_url = user_results["pictures"]["sizes"][1]["link"]
+    videos = video_results["data"]
+    return videos, uid, provider, username, avatar_url
+  end
+
+  def self.tweet_to_story(tweet, subscription)
```

Comment:
Ah, okay, so you did extracted out the 'storify' methods! You _could_ still have a private method that called the appropriate method based on the type of content, but it might just result in a long conditional, and be more trouble than it's worth!